std-bits/base/src/main/java/org/enso/base/net/http/UrlencodedBodyBuilder.java

```diff
@@ -12,17 +12,27 @@
 public final class UrlencodedBodyBuilder {
 
   private final ArrayList<String> parts = new ArrayList<>();
+  private String contents = null;
 
   /**
    * Create HTTP body publisher for an url-encoded form data.
    *
    * @return the body publisher.
    */
   public HttpRequest.BodyPublisher build() {
-    String contents = String.join("&", parts);
+    contents = String.join("&", parts);
     return HttpRequest.BodyPublishers.ofString(contents);
   }
 
+  /**
+   * Get the contents of the form data.
+   *
+   * @return the contents.
+   */
+  public String getContents() {
+    return contents;
+  }
```

Comment:
Should we add a `null` check here?

```suggestion
  public String getContents() {
  	if (contents == null) { throw new IllegalStateException("getContents should be called after build"); }
    return contents;
  }
```